clients/venice-push-job/src/main/java/com/linkedin/venice/pushjob/VenicePushJobConstants.java

```diff
@@ -1,11 +1,13 @@
-package com.linkedin.venice.hadoop;
+package com.linkedin.venice.pushjob;
```

Comment:
Can we move it to `com.linkedin.venice.common`?

Reply:
It's now renamed to vpj